frontend/src/Routes/Programs.tsx

```diff
@@ -20,6 +20,22 @@ import ProgramClassManagement from '@/Pages/ProgramClassManagement';
 import ClassLayout from '@/Components/ClassLayout';
 import Error from '@/Pages/Error';
 import { Navigate } from 'react-router-dom';
+import ResidentOverview from '@/Pages/ResidentOverview';
+
+export const ProgramRoutes = DeclareAuthenticatedRoutes(
+    [
+        {
+            path: 'programs-residents',
+            element: <ResidentOverview />,
+            handle: {
+                title: 'Current Enrollments',
```

Comment:
@corypride This has more than "Current Enrollments". Do you think we should call it "My Programs" or "Programs"?